Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -347,26 +347,25 @@ void GameList::ShowContextMenu(const QPoint&)
 
     menu->addSeparator();
 
-    auto* model = Settings::Instance().GetGameListModel();
-
     auto* tags_menu = menu->addMenu(tr("Tags"));
 
     auto path = game->GetFilePath();
-    auto game_tags = model->GetGameTags(path);
+    auto game_tags = m_model->GetGameTags(path);
 
-    for (const auto& tag : model->GetAllTags())
+    for (const auto& tag : m_model->GetAllTags())
     {
       auto* tag_action = tags_menu->addAction(tag);
 
       tag_action->setCheckable(true);
       tag_action->setChecked(game_tags.contains(tag));
 
-      connect(tag_action, &QAction::toggled, this, [this, path, tag, model](bool checked) {
-        if (!checked)
-          model->RemoveGameTag(path, tag);
-        else
-          model->AddGameTag(path, tag);
-      });
+      connect(tag_action, &QAction::toggled, this,
+              [ this, path, tag, model = m_model ](bool checked) {
```

Comment:
No need to capture `m_model` in a separate parameter. `this` is already available, `m_model` can be used instead.